staging/src/k8s.io/component-base/metrics/prometheus/restclient/metrics.go

```diff
@@ -111,13 +110,14 @@ func init() {
 	legacyregistry.MustRegister(requestResult)
 	legacyregistry.RawMustRegister(execPluginCertTTL)
 	legacyregistry.MustRegister(execPluginCertRotation)
+	/* TODO - temporary disable this for metrics migration was huge
```

Comment:
a task should be filed to track the effort to put this back.

Reply:
There is an existing PR for this task: https://github.com/CentaurusInfra/arktos/pull/980